docs/bug_report.md

```diff
@@ -9,10 +9,6 @@ assignees: kreyren nntoan
 
 ---
 
-**I'm submitting a ...**  (check one with "x")
-- [x] bug report
-- [ ] feature request
```

Comment:
Previous changes should make this format obsolete since we are using labels and `BUG: ` prefix